code/modules/surgery/surgery.dm

```diff
@@ -20,6 +20,9 @@
 	var/requires_tech = FALSE										//handles techweb-oriented surgeries, previously restricted to the /advanced subtype (You still need to add designs)
 	var/replaced_by													//type; doesn't show up if this type exists. Set to /datum/surgery if you want to hide a "base" surgery (useful for typing parents IE healing.dm just make sure to null it out again)
 	var/failed_step = FALSE											//used for bypassing the 'poke on help intent' on failing a surgery step and forcing the doctor to damage the patient
+	var/abductor_surgery_blacklist = list(/datum/surgery/advanced/necrotic_revival)
+	//Surgeries in this list aren't automatically available to those with the ABDUCTOR_SURGEON trait
+	//However, they can still be used by them if they meet the normal requirements to access the surgery
```

Comment:
it just should be `TRUE` or `FALSE` instead of being a list.
and the value in `/datum/surgery/advanced/necrotic_revival` just should be TRUE to blacklist.

Reply:
Addressed